db/controllers/controllers.js

```diff
@@ -55,3 +56,18 @@ exports.getAllArticles = (req,res,next)=>{
 }
 
 
+exports.patchArticleById =(req,res,next)=>{
+    
+    const patchData=req.body;
+    const {article_id}=req.params;
+    updateArticleById(patchData.inc_votes,article_id)
+    .then((article)=>{
+        res.status(201).send({article:article});
+    }).catch((err)=>{
+        next(err)
+    })
+}
+
```

Comment:
Good. If we're sending back an object with the same key and value name do we need to add both?